src/database/event-db.ts

```diff
@@ -63,13 +63,13 @@ export class Event {
     @prop({ default: 0 })
     points?: number;
 
-    @prop({ required: false })
+    @prop({ required: true })
     isPrivate?: boolean;
```

Comment:
if you set this required to be true, then remove the ?: from isPrivate, or add a default